docs/internal-developers/testing/releases/README.md

```diff
@@ -141,6 +141,7 @@ Every release includes specific testing instructions for new features and bug fi
     -   [10.0.1](./1001.md)
     -   [10.0.2](./1002.md)
     -   [10.0.3](./1003.md)
+-   [10.1.0](./1010.md)
```

Comment:
Indentation looks off - I don't know if that's an automated change or not, but better to mention it.

Reply:
Yes, it looks off, but that's happening because the 3 lines above are minor versions. The major versions has no indentation, like we can see here:

<img width="246" alt="image" src="https://user-images.githubusercontent.com/20469356/234397310-51fd76bc-37ed-4059-8c12-4057f532cf7a.png">